config/session.go

```diff
@@ -25,18 +25,61 @@ func init() {
 		// to immediately expire on the browser closing, set that option.
```

Comment:
Please update the comment:

<img width="544" alt="image" src="https://github.com/goravel/goravel/assets/24771476/35db20d9-0e97-4742-8c7b-2e1fb45c88bf">